### PR TITLE
add liveRadio e2e tests

### PR DESCRIPTION
**Pull request by jroebu14**
Resolves partially https://github.com/bbc/simorgh/issues/2989

**Overall change:** Add e2e tests for live radio pages

**Code changes:**

- added common, amp and canonical e2e tests

The config that enables these tests to be run is here https://github.com/bbc/simorgh/pull/3280

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/support/serviceHasPageType/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import pathSatisfies from 'ramda/src/pathSatisfies';
```
Comment (benjaminhobbs) [inline cypress/support/serviceHasPageType/index.js]: pls rmv ramda use

Comment (jamesbhobbs) [inline cypress/support/serviceHasPageType/index.js]: pls rmv ramda use

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]

File: cypress/integration/pages/liveRadio.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+            cy.title().should(
+              'eq',
+              `${name} - ${appConfig[service].brandName}`,
+            );
+          },
+        );
+      });
+    });
+
+    // will be addressed by https://github.com/bbc/simorgh/issues/2750
```
Comment (ryanmccombe) [inline cypress/integration/pages/liveRadio.js:42]: Is this comment still true?  that issue is closed now

File: cypress/integration/pages/liveRadio.js [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    describe('Scripts', () => {
+      it('should only have expected bundle script tags', () => {
+        cy.hasExpectedJsBundles(envConfig.assetOrigin, service);
+      });
+    });
+  });
+
+const runCanonicalTests = ({ service }) =>
+  describe('Canonical Tests', () => {
+    // will be addressed by https://github.com/bbc/simorgh/pull/2971
```
Comment (ryanmccombe) [inline cypress/integration/pages/liveRadio.js]:
```suggestion
    // will be addressed by https://github.com/bbc/simorgh/issues/3324
```

**Review by ryanmccombe — APPROVED** [review]

File: cypress/integration/pages/liveRadio.js (hunk 3 above)
Comment (jroebu14) [reply cypress/integration/pages/liveRadio.js]: good point I'll unskip those tests and see what happens!

**Comment by jamesbhobbs** [comment]
@jroebu14 there is something either wrong in this, or something I'm missing that this PR does. Specifically the CI builds seem to be false positives. Search for this cypress tests "should have one visible navigation with a skiplink to h1" it appears on latest branch and not on this one. Somehow this cuts them all. I've opened this PR https://github.com/bbc/simorgh/pull/3332 to highlight the issue better (see its failed builds), this PR if merged into this branch would cut changes to only adding the liveRadio.js file. Presuming you do this (which you should as all other tests now pass in that branch), you should investigate why the aforementioned test is failing in my PR and totally absent in this one.

**Comment by jroebu14** [comment]
@benjaminhobbs this PR removes that test https://github.com/bbc/simorgh/commit/ace7337a6db87a6b18dba075ca499381cc877d15#diff-6a133ac9ddaa5ea2078f57fc5c2567e1

**Review by jamesbhobbs — APPROVED** [review]

**Pull request merged**